src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
```

Comment:
The variable name 'tempValueRobot' could be more descriptive. Consider renaming it to 'robotValues'.